### PR TITLE
Update manage-device-tags.md

### PR DESCRIPTION
**Pull request by flinchbot**
2 minor grammatical changes at the top of the article.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/devices/manage-device-tags.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ appliesto:
...
 
 You can add a device tag to more than one device type. However, when you open a device pane in the admin center, only the devices of that type are returned. For example, you can assign the "Corporate" tag to both phones and Teams Rooms devices. If you search for the "Corporate" tag while in **Devices** > **Phones**, only phones are returned. Similarly, if you search for the "Corporate" tag in **Devices** > **Teams Rooms**, only Teams Rooms devices are returned.
 
 To manage device tags, you need to be a Global admin, Teams Service admin, or Teams Device admin. For more information about admin roles, see [Use Microsoft Teams administrator roles to manage Teams](../using-admin-roles.md).
 
 > [!IMPORTANT]
-> Device tags are assigned to the resource account that's logged into a device. If you sign a resource account out of one device and then use it to sign into another devices, the device tags are applied to new device.
+> Device tags are assigned to the resource account that's logged into a device. If you sign a resource account out of one device and then use it to sign into another device, the device tags are applied to new device.
```
Comment (JohanFreelancer9) [inline Teams/devices/manage-device-tags.md:30]:
```suggestion
> Device tags are assigned to the resource account that's logged into a device. If you sign a resource account out of one device and then use it to sign in to another device, the device tags are applied to the new device.
```

**Comment by scanum** [comment]
@flinchbot - Could you please make the changes requested? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**